Client/src/Components/MonitorTable/index.jsx

```diff
@@ -93,20 +94,15 @@ const Status = ({ params }) => {
  * @returns {React.Component} Returns a table with the monitor data.
  */
 const MonitorTable = ({ monitors = [] }) => {
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = useState(5);
+  const [page, setPage] = useState(1);
```

Comment:
This is a bit confusing as the reference pagination internally uses 0 based indexing, but I assume this is offset to show the user 1 based indexing.

I think a less confusing solution would be to leave the actual implementation as 0 based and move the offset to the rendering:

```
const [page, setPage] = useState(0);
...
const mappedRows = monitors.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
````

And then in the pagination component:

```
page={page + 1}
onChange={(event, value) => handleChangePage(event, value - 1)}
```

This is clearer to me since the pagination component expects a 0 based index.

Reply:
Yeah you're correct, it was done this way to simplify the logic calculating the offset, but I see where you're coming from. I'll add the fix 👍